cli/src/main/resources/help/shutdown.adoc

```diff
@@ -16,13 +15,29 @@ SYNOPSIS
 *shutdown cluster*
 
 
+USAGE CONSIDERATIONS
+--------------------
+*shutdown server*
+
+To prevent data loss, you should wait for any rebalancing operations to complete before stopping server instances.
+
+*shutdown cluster*
+
+When you gracefully stop a cluster, its state is persisted to the data folder for each node.
+
+Clusters can restart and be fully operational after shutdown only if the same nodes rejoin with the same cluster state that was persisted during shutdown.
+
+
 EXAMPLES
 --------
-`shutdown server my_server01` +
-Stops the server with hostname `my_server01`.
+`shutdown server` +
+Stops the server to which the CLI is connected.
+
+`shutdown server my_server01 my_server02` +
+Stops the servers with hostnames `my_server01` and `my_server02`.
 
 `shutdown cluster` +
-Performs an orderly shutdown of all servers joined to the cluster.
+Stops all nodes and persists cluster state.
```

Comment:
I would reverse the sentence and use "server" instead of "node"; something like:
Persists the cluster state and stops all servers.

Reply:
@pruivo How about "Stores cluster state, persists entries if you use a cache store, and stops all nodes."?

When we mean cluster state here, it's the global and cache state under the `data` folder. I think I got a bit mixed up with the Operator case that persists data during shutdown. I blame holiday brain.

Reply:
I was referring "cluster state" as everything: caches, topology, entries :) 
> @pruivo How about "Stores cluster state, persists entries if you use a cache store, and stops all nodes."?

Sounds good!